addons/web/static/src/xml/graph.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <div t-name="web.GraphOwlRenderer" owl="1">
+        <div class="o_graph_renderer">
+            <label t-if="props.title"><t t-esc="props.title"/></label>
+            <t t-if="noContentHelperData.show" t-call="web.NoContentHelper">
+                <t t-set="title" t-value="noContentHelperData.title"/>
+                <t t-set="description" t-value="noContentHelperData.description"/>
+            </t>
+            <t t-else="">
+                <t t-if="props.isEmbedded">
```

Comment:
instead of testing isEmbedded, you should use
a conditional style attribute on the canvas node
```
t-attf-style="#{props.isEmbedded ? '' : 'position: absolute;'}
```

Reply:
Actually: 
`t-attf-style="{{props.isEmbedded ? '' : 'position: absolute;'}}"`